tests/model_registry/rbac/conftest.py

```diff
@@ -405,3 +420,139 @@ def created_role_binding_user(
         subjects_kind="User",
         subjects_name=test_idp_user.username,
     )
+
+
+# =============================================================================
+# RESOURCE FIXTURES PARMETRIZED
+# =============================================================================
```

Comment:
why are you moving the fixture from tests/conftest.py to here?

Reply:
To move them closer to where they are being used. Only rbac tests uses these fixtures, so they should be under rbac/conftest.py.

Reply:
long run we should use only the parametrized fixtures and remove the default one, don't we?

Reply:
When we do, then they would be moved to right place. For now they should be where they are being used.